recipes/dataprep/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "dataprep" %}
+{% set version = "0.2.10a0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: dd74c71bbeb9faf10249dc8d20a5fe0f16551bf555c7ddcf593f8118141d1421
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  build:
+    - pip
+    - python >=3.6
+    - poetry >=1
+  run:
+    - python >=3.6
+    - bokeh >=2.0,<2.1
+    - dask[complete] >=2.13,<2.14
+    - holoviews >=1.13,<1.14
+    - jinja2 >=2.11,<2.12
+    - jsonpath-ng >=1.5.1,<2.0.0
+    - jsonschema >=3.2,<3.3
+    - lxml >=4.5,<4.6
+    - nltk >=3.5,<4.0
+    - numpy >=1.18,<1.19
+    - pandas >=1.0,<1.1
+    - pillow >=7.1.2,<8.0.0
+    - requests >=2.23,<2.24
+    - scipy >=1.4,<1.5
+    - tornado ==5.0.2
```

Comment:
Does this really need to be pinned so tightly?

Reply:
Yep, this is due to https://github.com/sfu-db/dataprep/issues/218.